src/index.js

```diff
@@ -639,6 +638,17 @@ module.exports = {
 			}
 		},
 
+		/**
+		 * Encode response data
+		 *
+		 * @param {HttpIncomingMessage} req
+		 * @param {HttpResponse} res
+		 * @param {any} data
+		 */
+		encodeResponse(req, res, data) {
+			return JSON.stringify(data, null, 2);
```

Comment:
The default code should be `JSON.stringify(data)`. The formatted JSON generates a big overhead and unnecessary for client codes.